src/Fakes/v1/YousignFaker.php

```diff
@@ -83,65 +85,65 @@ public function __construct()
         /** @var string $url */
         $url = Str::finish(Config::get(YousignClientServiceProvider::CONFIG_KEY . '.url'), Soa::URL_SEPARATOR);
 
-        Http::fake(
-            [
-                $url . Yousign::SIGNATURE_REQUESTS_BASE_URL => Http::response(
-                    SignatureRequestResponse::factory()
-                                            ->make()
-                                            ->toArray(),
-                    Response::HTTP_CREATED
-                ),
-                $url . Yousign::SIGNATURE_REQUESTS_BASE_URL . '/*/' . Yousign::DOCUMENT_URL => Http::response(
-                    UploadDocumentResponse::factory()
-                                          ->make()
-                                          ->toArray(),
-                ),
-                $url . Yousign::SIGNATURE_REQUESTS_BASE_URL . '/*/' . Yousign::ADD_CONSENT_URL => Http::response(
-                    AddConsentResponse::factory()
+        $fakers = [
```

Comment:
phpdoc (se la vuoi usare). Non sarebbe strettamente necessaria ma effettivamente mettere un array merge con dentro tutta questa pappardella diventa difficile da leggere.
```suggestion
        $fakes = [
```